googleapiclient/discovery.py

```diff
@@ -201,10 +201,8 @@ def build(serviceName,
   if resp.status >= 400:
     raise HttpError(resp, content, uri=requested_url)
 
-  try:
+  if isinstance(content, bytes):
     content = content.decode('utf-8')
-  except AttributeError:
-    pass
```

Comment:
You can change it into an assert if you think it makes more sense.  I don't think it does, though, as you're anyway turning the input into a unicode string and could just as well accept an already-unicode input.  But in any case I'd like to get rid of the current, broken, code.